tools/keycloak/sync/realm.json

```diff
@@ -556,4 +556,4 @@
       }
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
?

Reply:
This auto-formatting changes. Last PR from Stanley removed the newline at end of file. He doesn't run prettier automatically like we do on each PR